types/core.d.ts

```diff
@@ -16,6 +16,10 @@ export interface TaskTracker {
     [key: number]: TaskInfo;
 }
 
+export interface Region {
```

Comment:
How does this relate to properties called "region" in a few places which are string? It would be great to use a clearer or more qualified name here.

Reply:
Sounds good, will change and then resolve. I will think of a name.

Reply:
Named it RegionTracker